modules/sdk/gradle-plugins-workspace/src/gradleTest/workspaceProdctInfoDefaultValue/build.gradle

```diff
@@ -0,0 +1,15 @@
+task runGradleTest
+
+runGradleTest {
+	doLast {
+		String tomcatVersion = null
+
+		try {
+			tomcatVersion = gradle.liferayWorkspace.appServerTomcatVersion
+		}
+		catch (Exception exception) {
+		}
+
+		assert tomcatVersion == "9.0.31"
```

Comment:
Can you also add asserts for the rest of the product defaults?  and also can you have another testt were you override all of the product defaults to make sure that `gradle.properties` can apply the override?